src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/DetailsPaneViewModel.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace Microsoft.CmdPal.UI.ViewModels;
+
+public partial class DetailsPaneViewModel : ObservableObject
```

Comment:
If all this is doing is encapsulating the other VM we probably don't need it?

Technically in the View layer, if we're just doing stuff with controls, we don't need to have ViewModels for them if it's just doing its own View-y type stuff.

In this case, since the Details Control may be part of multiple different pages (or the shell itself), it's probably find to just pass it the `DetailsViewModel` directly that understands the notion of 'details' or what not in the context of the displayed item.

I'm probably not doing the best of explaining these concepts atm... but maybe that makes a bit of sense?

I need to grab lunch, but will look at things a bit more closely after, and happy to answer any questions if my statements above have caused confusion...

Reply:
I had this here _in case_ we wanted to have anything else in the details pane, like a "collapse" button, but you know what? That just belongs in the ShellPage doesnt it. 

Yea I suppose it doesn't need its own dedicated control, does it. The XAML encapsulation was nice though - the ShellPage is so tidy 😆